con-master/src/main/java/ba/codecentric/master/controller/RoomTypeController.java

```diff
@@ -0,0 +1,32 @@
+package ba.codecentric.master.controller;
+
+import ba.codecentric.base.domain.RoomType;
+import ba.codecentric.base.service.ImageService;
+import ba.codecentric.base.service.RoomTypeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.Valid;
+
+@RestController
+public class RoomTypeController {
+
+    private RoomTypeService roomTypeService;
+    private ImageService imageService;
+
+    @Autowired
+    public RoomTypeController(RoomTypeService roomTypeService, ImageService imageService) {
+        this.roomTypeService = roomTypeService;
+        this.imageService = imageService;
+    }
+
+    @RequestMapping(value = "/roomtype", method = RequestMethod.POST, consumes = {"multipart/form-data"})
+    public RoomType saveRoom(@RequestPart("image") MultipartFile image, @Valid @RequestPart("roomType") RoomType roomType) {
+        imageService.saveImage(image);
```

Comment:
Yes, and you can use notification messages which we already used for backend validation.